graspy/align/seedless_procrustes.py

```diff
@@ -0,0 +1,328 @@
+    matrix. Unlike OrthogonalProcrustes, this does not use a matching between
+    entries. It can even be used in the settings when the two datasets do not
+    have the same number of entries.
+
+    In graph setting, it is used to align the embeddings of two different
+    graphs, when it requires some simultaneous inference task, for example,
+    inside of the test for the equivalence of the latent distributions.
+
+    In essence, it aims to simultaneously obtain a, not necessarily 1-to-1,
+    correspondance between the vertices of the two data sets, and the
+    orthogonal matrix alignment matrix. It does via a sequence of alternating
+    steps, similarly to a regular expectation-maximization procedures. The step
+    of obtaining an assignment ("E-step") is done by solving an optimal
+    transport problem via Sinkhorn algorithm, whereas obtaining an orthogonal
+    alignment matrix ("M-step") is done via regular orthogonal procurstes.
+
+    Parameters
+    ----------
+        optimal_transport_lambda : float (default=0.1), optional
+            Regularization term of the Sinkhorn optimal transport algorithm.
+
+        optimal_transport_eps : float (default=0.01), optional
+            Tolerance parameter for the each Sinkhorn optimal transport
+            algorithm. I.e. tolerance for each "E-step".
+
+        optimal_transport_num_reps : int (default=1000), optional
+            Number of repetitions in each iteration of the iterative optimal
+            transport problem. I.e. maximum number of repetitions in each
+            "E-step".
+
+        iterative_eps : float (default=0), optional
+            Tolerance of the iterative optimal transport problem. I.e.
+            tolerance of the whole "EM" algorithm. Zero by default, which means
+            that the algorithm performs all `iterative_num_reps` number of
+            steps.
+
+        iterative_num_reps : int (default=100), optional
+            Number of reps in each iteration of the iterative optimal transport
+            problem. I.e. maxumum number of total iterations the whole "EM"
+            algorithm.
+
+        init : string, {"2d" (default), "sign_flips", "custom"}, optional
+
+            - "2d"
+                uses 2^d different initiazlizations, where d is the dimension.
+                specifically, uses all possible matrices with all entries real
+                and diagonal entries having magnitude 1 and 0s everywehre else.
+                for example, for d=2, tries [[1, 0], [0, 1]], [[1, 0], [0,
+                -1]], [[-1, 0], [0, 1]], and [[-1, 0], [0, -1]]. picks the best
+                one based on the value of the objective function.
+            - 'sign_flips'
+                for the first dimension, if two datasets have medians with
+                varying signs, flips all signs along this dimension for the
+                first dataset. then initializes to an identity.
```

Comment:
"Initial alignment done by making the median value in each dimension have the same sign"

Reply:
i like this concise way. i think what i have above is archaic phrasing from the time i was treating intialization matrix a little different. will change.

Reply:
addressed.